openapi-contracts/src/main/com/sinch/sdk/domains/voice/models/dto/v1/SvamlActionConnectConfDto.java

```diff
@@ -32,6 +34,39 @@
 @JsonFilter("uninitializedFilter")
 @JsonInclude(value = JsonInclude.Include.CUSTOM)
 public class SvamlActionConnectConfDto {
+  /** The name property. Must have the value &#x60;connectConf&#x60;. */
+  public enum NameEnum {
```

Comment:
Why do you need this enum? Is it because you have zero-trust to the server?

Reply:
Not a zero trust purpose but wish to have a "in specification we trust"
Purpose is to get this value and use value coming the the (OAS) specs and not duplicate expected hard coded value in sources.
Currently the expected value was describe from specs a default value. By adding it as an enum set with a single value; generator create a variable we can use from sources without duplication and hard coded "things"
I anything change from specs about the name: it will break compilation
If change is only for the value: no compilation issue nor at runtime: the expected value will be used without modification from sources